### PR TITLE
Adding tests for Tag Name parse errors

### PR DESCRIPTION
**Pull request by diervo**
- Added unexpected-eof-character error
- Added unexepceted-null-character error

### CODE REVIEW

File: tree-construction/webkit01.dat [hunk 1]
```diff
@@ -32,8 +32,9 @@ Test
 #data
 <di
 #errors
-(1,3): eof-in-tag-name
 (1,3): expected-doctype-but-got-eof
+#new-errors
```
Comment (inikulin) [inline tree-construction/webkit01.dat]: Don't remove old errors yet. They could be still used by implementers. We'll remove them all in one take once we have tree construction stage errors done.

**Review by inikulin — CHANGES_REQUESTED** [review]

**Comment by diervo** [comment]
@inikulin Added the error back, sorry for the slow pace, Im squeezing minutes in between meetings, will do a big chunk over the weekend :)

**Comment by inikulin** [comment]
@diervo No worries! I might be out of computer at the weekend, so sorry in advance if I'll be out of reach till Tuesday.

**Comment by diervo** [comment]
@inikulin Will try to do as much as I can in the meantime. Have fun!

**Pull request merged**